jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/udp/bsm/BsmReceiver.java

```diff
@@ -14,19 +16,17 @@ public class BsmReceiver extends AbstractUdpReceiverPublisher {
 
    private static Logger logger = LoggerFactory.getLogger(BsmReceiver.class);
 
-   private StringPublisher bsmPublisher;
+   private final StringPublisher bsmPublisher;
 
    @Autowired
-   public BsmReceiver(OdeProperties odeProps) {
-      this(odeProps, odeProps.getBsmReceiverPort(), odeProps.getBsmBufferSize());
-
-      this.bsmPublisher = new StringPublisher(odeProps);
```

Comment:
Note: you will see that I removed variations of this line in all `*Receiver.java` classes.  This was an unnecessary creation of a new StringPublisher. The publisher is already created in the constructor called by `this()`.  I did notice that by removing the duplicate creation of the publisher the overloaded constructors are unnecessary (in this iteration).  I chose to leave them alone because we may need them for future iterations of the OdeProperties migration. The duplication isn't harming anything at the moment, so I left them in (in violation of the YAGNI principle) for simplicity and reduction in the size of the change set. They will be removed in the next iteration if I find there still is no use for the overloaded constructors